docs/guides/latest-updates.mdx

```diff
@@ -14,24 +14,22 @@ Keep up with the latest and greatest from Qiskit and IBM Quantum&trade;! Gathere
 
 {/* release summaries written by product owners */}
 
-### Qiskit SDK v1.2.0
+### Qiskit SDK v1.3.0
 
 *To see the release notes for all versions, visit the [Qiskit SDK release notes](/api/qiskit/release-notes).*
 
-Qiskit 1.2 (stable release) is now available on [PyPI](https://pypi.org/project/qiskit/1.2.0/) and [GitHub](https://github.com/Qiskit/qiskit/releases/tag/1.2.0). Thanks to the maintainers and open-source contributors!
+The latest Qiskit version, 1.3, has just been released on [GitHub](https://github.com/Qiskit/qiskit/releases/tag/1.3.0) and [PyPI](https://pypi.org/project/qiskit/). The main changes include the following:
 
-- Many of the core structures have been moved to Rust, allowing for significant speedups mostly in the construction, manipulation (including copy and parameter assignment), and synthesis of circuits.
-- Better compiler quality due to the addition of unitary peephole optimization (for optimization levels 2 and 3) and improvements in Sabre.
-- [`generate_preset_pass_manager`](/api/qiskit/transpiler_preset#generate_preset_pass_manager) importance highlight, which is now importable from the top level.
+- We improved performance by porting additional key components to Rust, including:
+  - Core data structures, such as `DAGCircuit` or `Target`
+  - Transpiler passes, including `BasisTranslator`, `CommutationAnalysis`, or `UnitarySynthesis`, making the transpilation around six times faster compared to v1.2.4
+  - Circuit library and synthesis routines, such as `PauliEvolutionGate` or `n_local`
 
-Important deprecations:
-- Primitive V1 reference implementations (like `qiskit.primitives.Estimator`) and tooling are deprecated in favor of [V2 primitives](/api/qiskit/primitives). The primitive V1 base classes remain available.
-- [`BackendV1`](/api/qiskit/qiskit.providers.BackendV1) is now deprecated in favor of [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2), including the deprecation of `Qobj`, `assemble`, and other scaffolding related to the old V1.
-- Support for Python 3.8 will be dropped in the next release, v1.3. Python 3.9 will then be the new minimal required version.
+- The circuit library is now faster and allows for better high-level optimization, in particular with context-aware updates to `HighLevelSynthesis`.
 
-Remember, because this a minor release, it will not introduce breaking changes for users moving from Qiskit SDK v1.0. Any deprecations announced with this release refer only to things that will now start throwing deprecation warnings. None of these will be removed until the release of Qiskit v2.0, pursuant to Qiskit’s official [deprecation policy](https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md#).
+- Python 3.13 is now officially supported. Python 3.9 is now the *minimal* supported version.
```

Comment:
Should this be?

```suggestion
- Python 3.13 is now officially supported. Python 3.9 is now the *minimum* supported version.
```

Reply:
I see both on the internet but I think I like minimum better!